# Window Properties
window_x: 0
window_y: 0
window_width: 1920
window_height: 1080

font_size: 32

# Colors
text: [240,240,240]
background: [0,0,0]
border: [0,0,0]
